controllers/auth/loginController.js

```diff
@@ -13,6 +13,15 @@ const { User } = models;
  * @class
  */
 class Login {
+  /**
+   *
+   * SignIn.
```

Comment:
Please check the conventions of comments from `articleController.js` file. Look at how it is documented as in some of the other files of this project.